lib/mazaryn_web/templates/layout/app.html.heex

```diff
@@ -2,4 +2,6 @@
   <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
   <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
   <%= @inner_content %>
+
+  <%= render("_footer.html", assigns) %>
```

Comment:
@75pollet what's the reason for this change?

I'm working on #78 and I was applying some styles on the footer depending on whether the user is on the homepage or videos pages, and I was using `conn` to determine that. With this change, I don't have access to `conn` anymore.

Reply:
@75pollet He is waiting you to resolve it